### ISSUE TITLE
[BN] Add props to explicitly allow freezing moving mean & variance

### ISSUE THREAD

**Issue by zhoonit** [nnstreamer/nntrainer#1402]
When training but freezing batchnorm layer. There would be a case to freeze mean and variance in batchnormalization layer, we must support way to do this.

cc: a.escudero@samsung.com

**Comment by kparichay** [comment]
@zhoonit when trainable to set to false, any reason for BN layer to not freeze the mean and variance? does tf/pytorch update the BN mean/variance when the layer is not trainable?

**Comment by zhoonit** [comment]
I think there was a discussion that no longer really remember :p I think it was more like our policy that was conforming tf1.  

Tensorflow 2.0 uses trainable to implicitly set the behavior.
Pytorch have an explicit option (track_running_stats )

I prefer the pytorch way.

**Comment by kparichay** [comment]
We cant have some variables following the PyTorch style and others following the Tensorflow style. We must choose 1.
